templates/fullstack/server/src/schema/schema.graphql

```diff
@@ -1,29 +1,87 @@
 ## NOTE: This schema was generated by Graphback and should not be changed manually
 
+"""Exposes a URL that specifies the behaviour of this scalar."""
+directive @specifiedBy(
+  """The URL that specifies the behaviour of this scalar."""
+  url: String!
+) on SCALAR
+
 """ @model """
 type Comment {
   id: ID!
   text: String
   description: String
 
-  """@manyToOne field: 'comments', key: 'noteId'"""
+  """@manyToOne(field: 'comments', key: 'noteId')"""
   note: Note
 }
 
-input CommentInput {
+input CommentFilter {
+  id: IDInput
+  text: StringInput
+  description: StringInput
+  noteId: IDInput
+  and: [CommentFilter!]
+  or: [CommentFilter!]
+  not: CommentFilter
+}
+
+type CommentResultList {
+  items: [Comment]!
+  offset: Int
+  limit: Int
+  count: Int
+}
+
+input CommentSubscriptionFilter {
+  id: ID
+  text: String
+  description: String
+}
+
+input CreateCommentInput {
   id: ID
   text: String
   description: String
   noteId: ID
 }
 
+input CreateNoteInput {
+  id: ID
+  title: String!
+  description: String
+}
+
+input IDInput {
```

Comment:
As you see graphback is now fully compatible with the https://graphqlcrud.org matching features of the frameworks like prisma and hasura. GraphqlCrud gives us ability to use many partner libraries that rely on generated schema/client side queries

Reply:
We still did not announced it (as still working some terms with partners and need to improve spec - it is very very rough at the moment)